x-pack/elastic-agent/pkg/agent/install/root_windows_test.go

```diff
@@ -0,0 +1,20 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build windows
+
+package install
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestHasRoot(t *testing.T) {
+	t.Run("check if user is admin", func(t *testing.T) {
+		result, err := HasRoot()
+		assert.NoError(t, err)
+		assert.True(t, result)
```

Comment:
Does this mean the tests run as root? I don't think that is always the case.

I run the tests locally sometimes, and they would not be ran as root.

Reply:
indeed, so a few options here, either we execute a PowerShell cmd to check and validate if the function returns the same, or we disable the test. Just checked with the automation team, the windows test in CI are always ran under admin so no issues there.

Reply:
how about just change it to check the error, and ignore the true or false result?

Reply:
related issue was here: https://github.com/elastic/beats/pull/21884/files

Reply:
or that